src/pages/Shop/SideFilterList/DropDownFilter/DropDownFilter.jsx

```diff
@@ -2,13 +2,29 @@ import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
 import Checkbox from '@material-ui/core/Checkbox';
 
-const DropDownFilter = props => {
+const DropDownFilter = ({
+  name,
+  getCategory,
+  id,
+  categoryName,
+  removeSelected,
+}) => {
+  const categoryFilterHandler = e => {
+    console.log(`e`, e);
+    const checked = e.target.checked;
+    if (checked) getCategory(id, categoryName);
+    else if (!checked) removeSelected(id, categoryName);
+  };
+
   return (
     <Drop>
       <div>
-        <Checkbox inputProps={{ 'aria-label': 'uncontrolled-checkbox' }} />
+        <Checkbox
+          onClick={e => categoryFilterHandler(e)}
```

Comment:
- event 는 인자로 자동으로 들어갑니다.
- 따라서 현재와 같이 별도의 함수를 한번 더 만들어줄 필요가 없어보입니다!
```suggestion
          onClick={categoryFilterHandler}
```

Reply:
수정했습니다.